docs: diagrama de blocos

RA-Salles opened the pull request:
# Mudanças
- Adiciona o diagrama de blocos do sistema
- Adiciona nova página para hospedar diagrama de blocos
- Cria pasta assets para guardar demais imagens
- Cria aviso para demais usuários da pasta seguirem a estrutura proposta

# Lista de verificação
- A documentação possui o linguajar apropriado para documentação interna
- A documentação criada ou alterada está clara
- A imagem é visível e provê entendimento do funcionamento dos componentes do sistema

# Observações
Aos demais membros que não tem o costume de utilizar o github de forma efetiva, 
eu considero esse formato como o mínimo necessário para uma pull request efetiva.
Utilizem a palavra chave "Closes" seguida do número da issue para linkar o pr com a issue.

Closes #12

RA-Salles committed:
DOCS: adicionando diagrama de blocos

Como os autores e log de alterações podem ser facilmente inspecionados pelo github, resolvi não adicionar esse pedaço do artefato.

aprxl commented:
Aguardo o desenvolvimento do esquemático do circuito para concluirmos a tarefa. Boto a issue em pausa.

Renurin reviewed (changes requested):
Cada seta incluída no diagrama deve conter uma breve descrição da sua função, como por exemplo barramento de controle da ESP32 para a ponte H. Além disso, as conexões de alimentação devem ser representadas em outras cores. Por último, resta a comunicação com o computador via wifi.

aprxl reviewed (changes requested):
Ficou faltando o módulo acelerômetro-giroscópio no diagrama, e não tem necessidade pra esse README.txt também 😄

RA-Salles commented:
Acabei de ler a revisão. Alterando o diagrama.

RA-Salles committed:
DOCS: reestruturação do diagrama de blocos

Adiciona o módulo de sensores;
adiciona legenda às setas;
versiona nome do diagrama;
leve refatoração do texto;
exclusão do readme da pasta assets (Deus permitirá que essa pasta fique organizada)

RA-Salles committed:
DOCS: atualização do diagrama de blocos

Adição das cores de conexão de componentes.
Adição de uma versão clara.

RA-Salles commented:
Diagrama refatorado na branch!

Renurin reviewed (approved):
Perfeito!

aprxl reviewed (changes requested):
Faltou adicionar o background da label `WIFI` para melhor legibilidade, favor mover o diagrama de `assets/arquitetura/blocos` para `assets` e dar um nome mais descritivo pros arquivos.

aprxl committed:
Renomeado arquivos e removido nesting desnecessário

aprxl committed:
Arrumado o link para imagem

aprxl commented:
Aprovado 👏

RA-Salles commented:
paaaah, agora eu vi. 
Hooooooo

RA-Salles commented:
Fechando a branch só pra não virar o infeliz espaguete de branches.
